static/xref/script.js

```diff
@@ -91,6 +91,9 @@ function getFormData() {
 async function onSubmit(event) {
   event.preventDefault();
   const data = getFormData();
+  if (data.term === "") {
```

Comment:
does it really crash? `""` is a valid enum value, right? IIRC, it was defined at 3 places

Reply:
Sorry, this was before I realized that I was missing PR 34 from the other repo.